GOVERNANCE.md

```diff
@@ -166,19 +166,18 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
+If the private discussion receives some support (in form of explicit comments or
+thumbs-up reactions) and no explicit block after a few days, add a comment in the
```

Comment:
A "few days" can mean different things to different people. Let's make it explicitly one week (7 days).